src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -246,6 +246,65 @@ public void GetFolderPath_OSX_NonEmptyFolderPaths(Environment.SpecialFolder fold
             }
         }
 
+        [Theory]
```

Comment:
I'll move it- I'm also failing as I need to update build tools.  https://github.com/dotnet/buildtools/pull/1022